content/docs/user-guide/gems/development/creating/_index.md

```diff
@@ -8,10 +8,10 @@ You can develop your own independent modules to distribute and build as Open 3D
 bundle of code and/or assets, a manifest file, a CMake build file, and an optional display icon for project configuration
 tools. This is all that's needed to ship a Gem.
 
-Although you can create a Gem manually by creating all of the files yourself, it's recommended to use the `o3de` tool:
+Although you can create a Gem manually by creating all of the files yourself, it's recommended to use the `o3de` tool located in the `<engine>/scripts` directory:
 
 ```cmd
-o3de create-gem -gp <path to create gem at>
+o3de.bat create-gem -gp <path to create gem at>
```

Comment:
Ideally we should be showing both Windows and Linux examples now using the `tabs` shortcode. (See `user-guide/project-config/cli-reference.md` for an example of its use with o3de.bat/o3de.sh.)

Reply:
Done